util/Migrator/DbScripts/2024-03-25_00_EnableAllOrgCollectionEnhancements.sql

```diff
@@ -0,0 +1,37 @@
+-- This script will enable collection enhancements for organizations that don't have Collection Enhancements enabled.
+
+-- Step 1: Insert into a temporary table, selecting a percentage of the rows for each distinct PlanType
```

Comment:
Don't think this comment applies as you are not doing a percentage

Reply:
Fixed it!